core/arch/x86_64/mm/core_mmu.c

```diff
@@ -2487,6 +2544,8 @@ void core_mmu_set_user_map(struct core_mmu_user_map *map)
 {
 	if (map == NULL)
 		clear_user_map();
```

Comment:
You can't clear user map here. Otherwise, you still need to recreate mapping table.